src/components/LikeButton/README.md

```diff
@@ -0,0 +1,47 @@
+## LikeButton
+An implementation of an LikeButton for TPAs,
+
+By default the LikeButton will pull the theme from the site (site colors),
+
+## Theme properties (All are optional)
+
+| propName   | propType | defaultValue | description |
+|------------|----------|--------------|-------------|
+| IconColor  | string   | none | The color of the icon border instead of the default color set by theme |
+| LabelColor  | string   | none | The color of the label instead of the default color set by theme |
+| LabelFont  | string   | none | The font of the label instead of the default font |
```

Comment:
I think the icon color shouldn't be exposed here.
And the same thing with the default values: https://zeroheight.com/7sjjzhgo2/p/05a744/t/7933bc

Reply:
In this case, when the LikeButton is selected icon color is always red, but user can still change the border of the icon (when it is not selected). That's why it is left here.
So you think it should be removed?